OpenRA.Game/Traits/World/Shroud.cs

```diff
@@ -92,7 +94,7 @@ public Shroud(Actor self, ShroudInfo info)
 			explored = new CellLayer<bool>(map);
 		}
 
-		void INotifyCreated.Created(Actor self)
+		public void Created(Actor self)
```

Comment:
Don't revert this although it is used inconsistently in the code base. https://github.com/OpenRA/OpenRA/pull/10143